README.md

```diff
@@ -239,6 +239,22 @@ This strategy assumes that each migration script in the `db` directory has a u
     ```
 
 
+# How to add or change an encrypted Travis environment variable
+
+Reference: https://docs.travis-ci.com/user/environment-variables/#encrypting-environment-variables
+
+1. Download the Travis CLI `brew install travis` or `gem install travis`
+
+1. cd to anywhere in this repo
+
+1. For a given `VAR_NAME` that you want to have value `secret_value`, run
+   `travis encrypt --pro VAR_NAME=secret_value`
+   which will ask for your GitHub username and password and then
+   output your encrypted variable
+
+1. Copy the encrypted variable into `.travis.yml`
+
+
```

Comment:
Maybe there is a reason but I'm wondering why are these all numbered 1?

Reply:
I think github markdown automatically increments the numbered bullets in the rendered rich text

Reply:
Oh cool I didn't know that, thanks! :)

Reply:
Yup, it's just a Markdown thing. You can use numbers that go up, it would work the same way. But for comparing changes in the file it's nicer to not have to go back and change the numbering later on because you added a new step before what was your step `1.`